src/main/webapp/app/store/AnnotationStore.ts

```diff
@@ -291,20 +292,15 @@ export class AnnotationStore {
   readonly geneNumber = remoteData<GeneNumber>({
     await: () => [this.gene],
     invoke: async () => {
-      return privateClient.utilsNumbersGeneGetUsingGET({
-        hugoSymbol: this.gene.result.hugoSymbol,
-      });
-    },
-    default: {
-      gene: DEFAULT_GENE,
-      alteration: 0,
-      highestSensitiveLevel: '',
-      highestResistanceLevel: '',
-      highestDiagnosticImplicationLevel: '',
-      highestPrognosticImplicationLevel: '',
-      highestFdaLevel: '',
-      tumorType: 0,
+      try {
+        return await privateClient.utilsNumbersGeneGetUsingGET({
+          hugoSymbol: this.gene.result.hugoSymbol,
+        });
+      } catch (e) {
+        return DEFAULT_GENE_NUMBER;
+      }
```

Comment:
Another option beside `try...catch` is to use `onError: () => { return DEFAULT_GENE_NUMBER }` ?